cnho_stables/chain.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cnhostables",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cnho.io/",
+  "pretty_name": "CNHO Stables",
+  "chain_id": "cnho_stables-1",
+  "bech32_prefix": "cnho",
+  "daemon_name": "cnho",
+  "node_home": "$HOME/.cnho",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ucnho",
+        "fixed_min_gas_price": 0.000005,
+        "low_gas_price": 0.000001,
+        "average_gas_price": 0.00001,
+        "high_gas_price": 0.025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ucnho"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/alashooinc/Chain_CNHO/",
+    "recommended_version": "v1",
+    "compatible_versions": [
+      "v1"
+    ],
+    "cosmos_sdk_version": "v0.46.7",
+    "cosmwasm_enabled": false,
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/alashooinc/Chain_CNHO/master/mainnet/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v1",
+        "recommended_version": "v1",
+        "compatible_versions": [
+          "v1"
+        ],
+        "cosmos_sdk_version": "v0.46.7",
+        "cosmwasm_enabled": false
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.svg"
+   
+  },
+  "description": "CNHO Stables Network is designed to facilitate the real world asset in Cosmos ecosystem.",
+  "peers": {
+    "seeds": [
+       {
+        "id": "aeb501bba5d51c3aaac3a4d495c5b38be61c4dc2",
+        "address": "159.138.232.248:26656",
+        "provider": "CNHO Stables"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "aeb501bba5d51c3aaac3a4d495c5b38be61c4dc2",
+        "address": "159.138.232.248:26656",
+        "provider": "CNHO Stables"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://rpc.cnho.io",
+        "provider": "CNHO Stables"
+      } 
+    ],
+    "rest": [
+      {
+        "address": "https://api.cnho.io",
+        "provider": "CNHO Stables"
+      } 
+    ],
+    "grpc": [
+      {
+        "address": "159.138.232.248:9090",
+        "provider": "CNHO Stables"
+      } 
+    ]
+  },
+  "explorers": [
+    {
+      "url": "https://explorer.cnho.io",
+      "tx_page": "https://explorer.cnho.io/transaction/${txHash}",
+      "account_page": "https://explorer.cnho.io/account/${accountAddress}"
+    }
+  ],
+  "keywords": [
+    "CNHO"
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
```

Comment:
The svg and png in the same image object must match. Please replace the white background of the png with a transparency layer (just as the svg has)

Reply:
And make sure the margins match